scripts/version-packages.mjs

```diff
@@ -8,6 +8,7 @@ import prettier from 'prettier'
 
 const __dirname = path.dirname(url.fileURLToPath(import.meta.url))
 const root = path.resolve(__dirname, '..')
+const version = process.argv[2] || null
```

Comment:
This just allows us to force a version without manually editing the versions. E.g.:
```
pnpm run version-packages 0.0.0-insiders.abc123
```

Reply:
ooh this is handy 💯

Reply:
Ohh nice

Reply:
I couldn't immediately find an option in pnpm to do that recursively, so thought to just add it to this little script :P